pkg/kernel/patches-5.10.x/0005-fget-Do-not-loop-with-rcu-lock-held.patch

```diff
@@ -1,22 +1,20 @@
-From 59ab1a96357ccc470294663e0ceeeaab0c0b7ddd Mon Sep 17 00:00:00 2001
+From 0f0f02452a0ee5eb07126acb8d6dfd53e8502d4e Mon Sep 17 00:00:00 2001
 From: Sergey Temerkhanov <s.temerkhanov@gmail.com>
 Date: Wed, 4 Nov 2020 04:16:57 -0500
-Subject: [PATCH] fget: Do not loop with rcu lock held
+Subject: [PATCH 05/12] fget: Do not loop with rcu lock held
 
 Do not loop wint the RCU lock held
-
-Signed-off-by: Sergey Temerkhanov <s.temerkhanov@gmail.com>
 ---
  fs/file.c | 34 ++++++++++++++++++++--------------
  1 file changed, 20 insertions(+), 14 deletions(-)
 
 diff --git a/fs/file.c b/fs/file.c
-index 5a0fa61504ff..c9ac0232df20 100644
+index 21c0893f2f1d..ab12f5fbcecb 100644
 --- a/fs/file.c
 +++ b/fs/file.c
-@@ -715,21 +715,27 @@ static struct file *__fget(unsigned int fd, fmode_t mask, unsigned int refs)
+@@ -821,21 +821,27 @@ static struct file *__fget_files(struct files_struct *files, unsigned int fd,
```

Comment:
Again -- seems like a material change

Reply:
I believe this is purely the difference in how patches are generated. See the section "Testing done".
I cloned linux twice, applied old patches to one folder, and new patches to the new folder, and check if there is any difference. There was no

Reply:
I think I know why it looks different. What you see here is the diff on diff. Looks like the old patches were generated agains different version of kernel, but still apply on newer?